amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/main-indexing.json

```diff
@@ -0,0 +1,12 @@
+{
+    "genome_fasta_url": "ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_31/GRCh38.primary_assembly.genome.fa.gz",
+    "genome_fasta_filename": "GRCh38.primary_assembly.genome.fa",
+    "annotation_gtf_url": "ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_31/gencode.v31.annotation.gtf.gz",
+    "annotation_gtf_filename": "gencode.v31.annotation.gtf",
+    "nthreads": 45,
+    "synapse_config": {
+        "class": "File",
+        "path": "./.synapseConfig"
+    },
+    "output_synapse_id": "syn20675525"
+}
```

Comment:
This is the args file to be used in `run-test-indexing.py`. All of these various run files will get re-organized a bit when the script refactor is done in SCIENCE-1170.